heron_wsgi/traincheck/traincheck.py

```diff
@@ -488,7 +494,7 @@ def TrainingRecordsAdmin(acct, exempt_pid,
            "CRS"."dtePassed" AS completed,
            "CRS"."strCompletionReport" AS course
     FROM "CRS"
-    WHERE "CRS"."strGroup" IN (:strGroup_1, :strGroup_2)
+    WHERE "CRS"."strGroup" IN (:strGroup_1, :strGroup_2, :strGroup_3, :strGroup_4, :strGroup_5, :strGroup_6, :strGroup_7, :strGroup_8)
```

Comment:
i spot checked some of the added course names; it looks like none of them exist in the CRS table.  for example:

```
mysql> select count(*) from CRS where strGroup = 'CITI Human Subjects Research';
+----------+
| count(*) |
+----------+
|        0 |
+----------+
```

so.. i don't see how these changes are going to fix the problem.